.github/ISSUE_TEMPLATE/regression_checklist.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Regression checklist
+about: Suggest an idea for this project
```

Comment:
Could we change this to "Checklist for smoke testing of release build." or something similar.

Reply:
done